### ISSUE TITLE
GD5 beta 39 doesn't work on windows 10

### ISSUE THREAD

**Issue by Oneil974** [4ian/GD#524]
Hello,

I have a black screen in GD5 Beta 39. It does not seem to work.

It just gives me a javascript error like this when I try to create a new project.

![gd5_bug_screen2](https://user-images.githubusercontent.com/11059074/41510488-9b97f00e-7276-11e8-8f8f-1756386a789a.png)

**Comment by 4ian** [comment]
I've tried successfully GD beta 42 on Windows 10 and I think other users are having it working too. Can you try again with the latest version? Uninstall GD completely first.

**Comment by Oneil974** [comment]
Sorry, maybe I downloaded the wrong release. I've downloaded it directly from the website last night.
Must I build it from sources? (Beta42)

**Comment by 4ian** [comment]
No it was released yesterday :) You can simply go on the website and download it again to have the latest version.

**Comment by Oneil974** [comment]
Same problem with the Beta42 version... :(
Impossible for me to open dev tools or debugg.
Maybe something installed on my Windows.
I'm going to inspect

**Comment by Oneil974** [comment]
Maybe there are some requirements?

**Comment by Oneil974** [comment]
Hey, I found the problem.

With my M11XR3 I have optimus technology (Nvidia + Intel Graphics) I must force run GD5 with Intel integrated graphic card.

**Comment by Oneil974** [comment]
Yes, the bug no more exist after Nvidia driver update. The bug occured after last W10 update.

**Comment by 4ian** [comment]
Cool! :)
